test/unit/helpers/people_helper_test.rb

```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class PersonHelperTest < ActionView::TestCase
+  include ApplicationHelper
+
+  test "it disambiguates names with trailing spaces" do
+    person_one = build(:person, forename: "Peter ", surname: "Jones")
```

Comment:
ah I see! Maybe instead of 'Vic' & 'Bob' it could be 'Peter Jones - Executive Director, Prisons' & 'Peter Jones - Director, Overseas Territories' to more closely match the result of `person.name_with_disambiguator`?